daemon/graphdriver/aufs/aufs.go

```diff
@@ -311,27 +310,16 @@ func (a *Driver) Remove(id string) error {
 
 	var retries int
 	for {
-		mounted, err := a.mounted(mountpoint)
-		if err != nil {
-			if os.IsNotExist(err) {
-				break
-			}
-			return err
-		}
-		if !mounted {
-			break
-		}
-
 		err = a.unmount(mountpoint)
```

Comment:
```suggestion
		err := a.unmount(mountpoint)
```

Reply:
Sorry for the noise, have pushed some intermediate (unfinished) version late last night... should be fixed now.